code/modules/paperwork/records/info/zayin.dm

```diff
@@ -71,7 +71,7 @@
 	abno_code = "D-01-145"
 	abno_info = list(
 		"When work with T-01-145 was completed, it started to speak.",
-		"If the agent working with T-01-145 listened to it's entire story, they received a buff for the corresponding attribute, lasting 5 minutes.")
+		"If the agent working with T-01-145 listened to it's entire story, everyone in the room received a buff for the corresponding attribute, lasting 5 minutes.")
```

Comment:
It's a 5 minute +15 stat buff we don't need to cram like 15 people into one room to give them all buffs.
I've said this much so far and already.